### DISCUSSION TITLE
I can't understand authelia can use TPM2.0 random generator?

### DISCUSSION THREAD

**Discussion by N0TScrooge** [authelia/authelia#8343] [Q&A] [answered]
Tell me where I can see

**Comment by james-d-elliott** [comment, marked as answer]
Not specifically, but it wouldn't be all that difficult to implement I don't think.

**Reply by N0TScrooge** [reply]
How I think it would be a good option to improve security

**Reply by james-d-elliott** [reply]
For what specific tasks would it improve it?

**Reply by N0TScrooge** [reply]
A random number hardware generator is always better than a pseudo-generator.
